daemon/api.go

```diff
@@ -1519,7 +1519,13 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 	}
 
 	gecos := fmt.Sprintf("%s,%s", createData.Email, v.OpenIDIdentifier)
-	if err := osutilAddExtraUser(v.Username, v.SSHKeys, gecos, createData.Sudoer); err != nil {
+	opts := &osutil.AddUserOptions{
+		SshKeys:       v.SSHKeys,
+		Gecos:         gecos,
+		Sudoer:        createData.Sudoer,
+		UseExtraUsers: !release.OnClassic,
```

Comment:
Since we need to tweak it due to the above, let's also s/UseExtraUsers/ExtraUsers/ please.